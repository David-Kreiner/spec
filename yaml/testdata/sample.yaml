pipeline:
  inputs:
    username:
      type: string
      default: ubuntu
    password:
      type: string

  clone:
    depth: 50

  env:
    GOOS: linux

  environment: prod

  stages:
  - steps:
    - go build
    - run: go build
    - run:
        container: golang
        script: go build
    - run:
        container:
          image: golang
          pull: if-not-exists
        script:
        - go build
        - go test

