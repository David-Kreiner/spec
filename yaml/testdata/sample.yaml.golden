pipeline:
  stages:
  - steps:
    - run:
        script:
        - go build
        - go test
